core/responsive/responsive.php

```diff
@@ -66,13 +66,15 @@ class Responsive {
 	 *
 	 * Retrieve the default responsive breakpoints.
 	 *
+	 * @return array Default breakpoints.
+	 * @deprecated 3.2.0
 	 * @since 1.0.0
 	 * @access public
 	 * @static
-	 *
-	 * @return array Default breakpoints.
 	 */
 	public static function get_default_breakpoints() {
+		Plugin::$instance->modules_manager->get_modules( 'dev-tools' )->deprecation->deprecated_function( __METHOD__, '3.2.0', 'Elementor\Core\Breakpoints\Manager::get_default_config()' );
```

Comment:
Please add the real replacement `Plugin::$instance...`

Reply:
Added by mistake.
I didn't have to add it though that is what appears in the deprecations file

Reply:
I didn't understand

Reply:
Sorry, I didn't understand what you meant by writing "real replacement"  this is what is written in the deprecations sheet.

Reply:
My mistake too, it's a static method that will work.